maintainers/maintainer-list.nix

```diff
@@ -82,6 +82,12 @@
     githubId = 882455;
     name = "Elliot Cameron";
   };
+  _414owen = {
```

Comment:
Please put this into a separate commit with the message ``maintainers: add _414owen``.